src/script.js

```diff
@@ -5,32 +5,46 @@
 //HTML Elements
   const container1 = document.getElementById("result1Box");
   const container2 = document.getElementById("result2Box");
+  const form = document.querySelector("form");
 
 //Functions
   //Get User Input
 
+  form.addEventListener("submit", async (event) => {
+    event.preventDefault();
+    const formData = new FormData(form);
+     await pullFromAPI(shakespeare, "POST", "application/JSON", formData, container1);
+  })
+
   //Get Stuff from API
     async function pullFromAPI(api, method, contentType, requestBody, targetDiv) {
-      const url = api + requestBody;
-      console.log(url);
-      const translatedText = fetch((url), {
+      
+      // Turn form text input into string and create full URL
+      const formDataObject = Object.fromEntries(requestBody.entries());
+      const url = api;
+      const queryString = new URLSearchParams(formDataObject).toString();
+      const fullURL = url + "?" + queryString;
+
+      console.log(fullURL);
+
+      const translatedText = fetch(fullURL, {
         method: method,
-        headers: {
-          "Content-Type": contentType,
-          "Host": host,
-        },
-        body: JSON.stringify(requestBody)
+        // headers: {
+        //   // "Content-Type": contentType,
+        //   "Host": host,
+        // },
+        body: JSON.stringify(formDataObject)
       })
       .then((response) => {
         if (!response.ok) {
           const error = new Error(response.status);
           throw error;
         }
-        response.JSON;
+        return response.JSON;
       })
-      .then((responseJSON) => JSON.parse(responseJSON))
+      // .then((responseJSON) => JSON.parse(responseJSON)) -- commented out as now happening further up
       .then((responseObject) => responseObject.contents.translated)
-      .catch(console.error(error))
```

Comment:
Bard lied to me, I did it without the nested parentheses too but I had lost my ability to reason (insomnia last night)